component/otelcol/exporter/prometheus/internal/convert/convert.go

```diff
@@ -61,6 +61,9 @@ type Options struct {
 	// IncludeScopeInfo includes the otel_scope_info metric and adds
 	// otel_scope_name and otel_scope_version labels to data points.
 	IncludeScopeInfo bool
+	// IncludeScopeLabels includes the otel_scope_name and otel_scope_version
+	// labels from the scope in the metrics.
+	IncludeScopeLabels bool
```

Comment:
Technically this is a breaking change. Previously, scope labels would be added when setting `include_scope_info`. Now, users have to set `include_scope_labels` separately.

Can we document that as a breaking change?